src/test/java/com/segment/analytics/android/integrations/amplitude/AmplitudeTest.java

```diff
@@ -246,7 +247,10 @@
   @Test public void reset() {
     integration.reset();
 
-    verify(amplitude).clearUserProperties();
+    verify(amplitude).regenerateDeviceId();
+
+    // Previously we called clearUserProperties() which was incorrect.
+    verify(amplitude, never()).clearUserProperties();
```

Comment:
Not strictly needed, but mostly to ensure we aren't still calling it. If i didn't have this, my test would still pass if I hadn't removed `clearUserProperties` from the integration. Also prevents a regression in the future.